common/src/BasePlugin.php

```diff
@@ -99,7 +99,15 @@ public function getPluginFile() {
         return dirname(dirname($reflect->getFilename())) . '/' . $this->getSlug() . '.php';
     }
 
+    public function getGtinMetaKey(): string {
```

Comment:
Since this is only used in WooCommerce-related code, I think it should be moved to the WooCommerce class.